rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -851,6 +851,37 @@ def test_dvehr_consent(self):
     ps_1 = self.send_get('Participant/%s/Summary' % participant_id_1)
     self.assertEquals('SUBMITTED_NO_CONSENT', ps_1['consentForDvElectronicHealthRecordsSharing'])
 
+  def testWithdrawThenPair(self):
+    participant_1 = self.send_post('Participant', {"providerLink": [self.provider_link]})
+    participant_id_1 = participant_1['participantId']
+    with FakeClock(TIME_1):
+      self.send_consent(participant_id_1)
+
+    ps_1 = self.send_get('Participant/%s/Summary' % participant_id_1)
+    self.assertEquals('NOT_WITHDRAWN', ps_1['withdrawalStatus'])
+    self.assertEquals('PITT', ps_1['awardee'])
+    self.assertIsNone(ps_1.get('withdrawalTime'))
+
+    with FakeClock(TIME_2):
+      participant_1['withdrawalStatus'] = 'NO_USE'
+      self.send_put('Participant/%s' % participant_id_1, participant_1,
+                    headers={ 'If-Match':'W/"1"'})
+
+    ps_1 = self.send_get('Participant/%s/Summary' % participant_id_1)
+    self.assertEquals('NO_USE', ps_1['withdrawalStatus'])
+    self.assertEquals('PITT', ps_1['awardee'])
+    self.assertEquals(TIME_2.isoformat(), ps_1.get('withdrawalTime'))
+
+    with FakeClock(TIME_3):
+      participant_1['providerLink'] = [self.az_provider_link]
```

Comment:
Does it seem right that we can repair a participant after a `NO_USE` withdrawal ?

Reply:
Yeah, that's a requirement... in particular, PTSC needs to be able to administratively re-pair withdrawn participants with TEST or whatever.